Templates/ObjC/Requests/EntityRequestBuilder.h.tt

```diff
@@ -193,8 +193,16 @@ foreach(var prop in entity.Properties.Where( prop => prop.Projection.Type is Odc
 
 <#
 }
+
+var allMethods = new List<OdcmMethod>();
 foreach(var func in entity.Methods)
 {
+   allMethods.Add(func);
+   allMethods.AddRange(func.Overloads);
+}  
+
+foreach(var func in allMethods)
```

Comment:
FYI, you should now (per [this](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/29/commits/ac0d2d10dafb8bc69e4a1039296ac1970c852e80) change) be able to reduce this code to `foreach(var func in entity.MethodsAndOverloads())`